Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ endif()
 # ITK --stop
 
 
+if(POLICY CMP0074)
```

Comment:
The same thing is done just below on line 14.

Reply:
Fixed.